### DISCUSSION TITLE
how to make a single variable update in model backword

### DISCUSSION THREAD

**Discussion by yaping1119** [google/flax#1384] [Q&A]
for example,  we can use  ' jax.value_and_grad(loss_fn, has_aux=True)(state.optimizer.target)' to update model variable, 
how can update single variable?

**Comment by jheek** [comment]
We recommend using [optax](https://github.com/deepmind/optax) for optimizing model parameters. Optax has a `mask`API  that allows you to update only a subset of the variables.
